virttest/env_process.py

```diff
@@ -856,7 +859,8 @@ def _call_fs_source_func():
                         vm.resume()
 
     def _call_check_image_func():
-        if params.get("skip_image_processing") == "yes":
+        vdpa_dev = params.get("storage_type") == "vhost-vdpa"
+        if params.get("skip_image_processing") == "yes" or vdpa_dev:
```

Comment:
Interesting, my comment was not seen in 3718..... Paste it here, again @nickzhq @YongxueHong 

storage_type is an image level param, while at line811, we did a global check there.  (The same comment for above change)

IMO, we can put the check in the function handling a single image, e.g. check_image, generate the QemuImg object after the check of 'check_image' param, then we can skip the check by setting check_image_image2 = no